src/utils/tokens/login.ts

```diff
@@ -0,0 +1,38 @@
+import { config } from '@config';
+import * as jwt from 'jsonwebtoken';
+
+const loginAlgorithm: jwt.Algorithm = 'ES256';
```

Comment:
Do we want to make this a config variable? I'm guessing no, but thought I'd pose the question anyways.

Reply:
no, the bot also isnt configurable, if we do it then we should do both